log4j-core/src/main/java/org/apache/logging/log4j/core/appender/SyslogAppender.java

```diff
@@ -425,6 +440,7 @@ protected SyslogAppender(final String name, final Layout<? extends Serializable>
                 .setCharsetName(charset)
                 .setExceptionPattern(exceptionPattern)
                 .setLoggerFields(loggerFields)
+                .setTimestampPrecision( timestampPrecision )
```

Comment:
No spaces around an argument, please follow the style of the file you're in!
`.setTimestampPrecision( timestampPrecision )` -> `.setTimestampPrecision(timestampPrecision)`

Reply:
I have fixed it with latest commit.